mixer/cmd/build.go

```diff
@@ -110,6 +112,144 @@ var buildBundlesCmd = &cobra.Command{
 	},
 }
 
+// buildOldFormatCmd is used to build the final version in the current format (the +10)
+// and ready the mix state for the new (+20) version to be built by a (possibly) newer mixer
+var buildOldFormatCmd = &cobra.Command{
+	Use:   "format-old",
+	Short: "Build the last version in the current format for the format bump",
+	Long:  `Build the last version in the current format for the format bump`,
```

Comment:
Not necessary yet, but before this gets released we really could use a good long-format description for this and `mixer build format-new`